### ISSUE TITLE
Wrong value on field "filter_field" on com_contact?

### ISSUE THREAD

**Issue by drmenzelit** [joomla/joomla-cms#8283]
com_contact -> Options -> List Layouts -> Filter Field: the button stays green when selecting "Hide"

administrator/components/com_contact/config.xml line 565 it should be 
`<option value="0">JHIDE</option> `
and not 
`<option value="hide">JHIDE</option>`

**Comment by infograf768** [comment]
Have you tested on staging?
https://github.com/joomla/joomla-cms/archive/staging.zip

This is what we have and will be released with 3.5.0
```
<field
			name="filter_field"
			type="radio"
			class="btn-group btn-group-yesno"
			default="1"
			description="JGLOBAL_FILTER_FIELD_DESC"
			label="JGLOBAL_FILTER_FIELD_LABEL"
			>
				<option value="1">JSHOW</option>
				<option value="0">JHIDE</option>		
		</field>
```

**Comment by drmenzelit** [comment]
Thanks for the hint! I'm learning how to contribute :-)
